models/ExhibitPage.php

```diff
@@ -405,4 +412,24 @@ public function setFixChildrenOnDelete($fix)
     {
         $this->_fixChildrenOnDelete = (bool) $fix;
     }
+
+    /**
+     * Get a property for display.
+     *
+     * @param string $property
+     * @return mixed
+     */
+    public function getProperty($property)
+    {
+        switch($property) {
+            case 'menu_title':
+                if(isset($this->short_title) && !empty($this->short_title)) {
```

Comment:
`!empty` should be sufficient by itself here, as it provides the same error-suppressing properties that `isset` does.